ui/app/templates/components/secret-edit-toolbar.hbs

```diff
@@ -21,13 +21,7 @@
   {{/unless}}
   <ToolbarActions>
     {{#if (eq @mode "show")}}
-      <SecretDeleteMenu
-        @modelForData={{@modelForData}}
-        @model={{@model}}
-        @isV2={{@isV2}}
-        @refresh={{action @editActions.refresh}}
-        @canReadSecretMetadata={{@canReadSecretMetadata}}
-      />
+      <SecretDeleteMenu @model={{@model}} />
```

Comment:
personally I'd prefer if we put the `ConfirmAction` here and moved the `handleDelete` action to this component, so we don't forget about it later. Or we can mark `SecretDeleteMenu` with another remove TODO

Reply:
Let's do the TODO. There is some conditional work that needs to be done in the secret-edit-toolbar for kv1 vs kv2 and this makes sense to do that in that PR. I can prioritize that PR to come next.